src/main/java/pib/Pib.java

```diff
@@ -0,0 +1,58 @@
+package pib;
+
+import pib.pibexception.PibException;
+import pib.utility.Parser;
+import pib.utility.Storage;
+import pib.utility.TaskList;
+import pib.utility.Ui;
+
+/**
+ * pib is a Personal Assistant Chat-bot that is able to keep track of tasks (CRUD) and deadlines
+ */
+public class Pib {
+
+    /**
+     * String to locate data file to load/save data
+     */
+    public static final String DATA_FILE_PATH = "data/data.txt";
+
+    private Ui ui;
+    private TaskList list;
```

Comment:
Perhaps consider changing the name to *tasks* so there's greater clarity of what this variable is supposed to be.